src/cpptrace.cpp

```diff
@@ -495,7 +495,7 @@ namespace cpptrace {
     namespace detail {
         std::atomic_bool absorb_trace_exceptions(true); // NOSONAR
         std::atomic_bool resolve_inlined_calls(true); // NOSONAR
-        std::atomic<enum cache_mode> cache_mode(cache_mode::prioritize_speed); // NOSONAR
+        std::atomic<cache_mode> cache_mode(cache_mode::prioritize_speed); // NOSONAR
```

Comment:
Minor nit, but maybe it'd be good to just rename the `cache_mode` variable while here instead of using `::cpptrace::cache_mode` later. This is my bad, I shouldn't have written the code like this in the first place. Maybe `current_cache_mode`?